src/Jackett.Common/Utils/Clients/HttpWebClient2.cs

```diff
@@ -33,19 +33,15 @@ public class HttpWebClient2 : WebClient
         protected static IWebProxy webProxy;
 
         [DebuggerNonUserCode] // avoid "Exception User-Unhandled" Visual Studio messages
-        public static bool ValidateCertificate(object sender, X509Certificate certificate, X509Chain chain, SslPolicyErrors sslPolicyErrors)
+        public static bool ValidateCertificate(HttpRequestMessage request, X509Certificate2 certificate, X509Chain chain, SslPolicyErrors sslPolicyErrors)
         {
-            if (sender.GetType() != typeof(HttpWebRequest))
-                return sslPolicyErrors == SslPolicyErrors.None;
```

Comment:
why you removed this validation?

Reply:
Because the sender is now a specific type `HttpRequestMessage` so we don't need to do type checking. See: https://github.com/Jackett/Jackett/blob/7b0adc7e45724b9601e23b689dfd2d6e364e7d6a/src/Jackett.Common/Utils/Clients/HttpWebClient2NetCore.cs#L35-L55

Reply:
It's ok